egs/iam/s5/local/chain/run_cnn_chainali_1b.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+# chainali_1b uses chain model for lattice instead of gmm-hmm model. It has more cnn layers as compared to 1a
```

Comment:
change this to "chainali_1b is as chainali_1a except it has 3 more cnn layers."
Then append the compare_wer.sh output (with the command) after adding a blank line